src/nav/full-stack-observability.yml

```diff
@@ -1654,6 +1654,9 @@ pages:
         path: /docs/auto-telemetry-pixie/get-started-auto-telemetry-pixie
       - title: Install Auto-telemetry with Pixie
         path: /docs/auto-telemetry-pixie/install-auto-telemetry-pixie
+        pages: 
+          - title: Manage your Auto-telemetry with Pixie data ingest
```

Comment:
at this point I believe you that Auto needs to be initial cap. but just in case...